sagemaker-debugger/tensorflow2/tensorflow2_keras_custom_container/src/tf_keras_resnet_byoc.py

```diff
@@ -0,0 +1,88 @@
+"""
+This script is a ResNet training script which uses Tensorflow's Keras interface.
+It has been orchestrated with SageMaker Debugger hooks to allow saving tensors during training.
```

Comment:
This script provides example of how to use sagemaker debugger if you use 
  1) your own custom container to run in sagemaker
  2) your own script outside sagemaker

Reply:
changed in the latest commit